src/Fixer/Operator/TernaryToNullCoalescingFixer.php

```diff
@@ -152,30 +152,37 @@ private function getMeaningfulSequence(Tokens $tokens, int $start, int $end): To
      */
     private function isHigherPrecedenceAssociativityOperator(Token $token): bool
     {
-        static $operatorsPerId = [
-            T_ARRAY_CAST => true,
-            T_BOOLEAN_AND => true,
-            T_BOOLEAN_OR => true,
-            T_BOOL_CAST => true,
-            T_COALESCE => true,
-            T_DEC => true,
-            T_DOUBLE_CAST => true,
-            T_INC => true,
-            T_INT_CAST => true,
-            T_IS_EQUAL => true,
-            T_IS_GREATER_OR_EQUAL => true,
-            T_IS_IDENTICAL => true,
-            T_IS_NOT_EQUAL => true,
-            T_IS_NOT_IDENTICAL => true,
-            T_IS_SMALLER_OR_EQUAL => true,
-            T_OBJECT_CAST => true,
-            T_POW => true,
-            T_SL => true,
-            T_SPACESHIP => true,
-            T_SR => true,
-            T_STRING_CAST => true,
-            T_UNSET_CAST => true,
-        ];
+        static $operatorsPerId;
+        if (null === $operatorsPerId) {
+            $operatorsPerId = [
+                T_ARRAY_CAST => true,
+                T_BOOLEAN_AND => true,
+                T_BOOLEAN_OR => true,
+                T_BOOL_CAST => true,
+                T_COALESCE => true,
+                T_DEC => true,
+                T_DOUBLE_CAST => true,
+                T_INC => true,
+                T_INT_CAST => true,
+                T_IS_EQUAL => true,
+                T_IS_GREATER_OR_EQUAL => true,
+                T_IS_IDENTICAL => true,
+                T_IS_NOT_EQUAL => true,
+                T_IS_NOT_IDENTICAL => true,
+                T_IS_SMALLER_OR_EQUAL => true,
+                T_OBJECT_CAST => true,
+                T_POW => true,
+                T_SL => true,
+                T_SPACESHIP => true,
+                T_SR => true,
+                T_STRING_CAST => true,
+                T_UNSET_CAST => true,
+            ];
+            if (\defined('T_AMPERSAND_FOLLOWED_BY_VAR_OR_VARARG')) { // @TODO: drop condition when PHP 8.1+ is required
+                $operatorsPerId[T_AMPERSAND_FOLLOWED_BY_VAR_OR_VARARG] = true;
+                $operatorsPerId[T_AMPERSAND_NOT_FOLLOWED_BY_VAR_OR_VARARG] = true;
+            }
+        }
```

Comment:
not needed, used `equalsAny` which uses `equals` which you fixed for `&`

Reply:
This one we do not need, removed.